write-fonts/src/tables/glyf.rs

```diff
@@ -427,6 +446,32 @@ mod tests {
         assert_eq!(orig_bytes.len(), bytes.len());
     }
 
+    #[test]
+    fn round_trip_multi_contour() {
+        let font = FontRef::new(test_data::test_fonts::VAZIRMATN_VAR).unwrap();
+        let loca = font.loca(None).unwrap();
+        let glyf = font.glyf().unwrap();
+        let read_glyf::Glyph::Simple(orig) = loca.get_glyf(GlyphId::new(1), &glyf).unwrap().unwrap() else { panic!("not a simple glyph") };
+        let orig_bytes = orig.offset_data();
+
+        let bezpath = simple_glyph_to_bezpath(&orig);
+
+        let ours = SimpleGlyph::from_kurbo(&bezpath).unwrap();
+        let bytes = crate::dump_table(&ours).unwrap();
+        let ours = read_glyf::SimpleGlyph::read(FontData::new(&bytes)).unwrap();
+
+        let our_points = ours.points().collect::<Vec<_>>();
+        let their_points = orig.points().collect::<Vec<_>>();
+        dbg!(
+            SimpleGlyphFlags::from_bits(1),
+            SimpleGlyphFlags::from_bits(9)
+        );
+        assert_eq!(our_points, their_points);
+        assert_eq!(orig.glyph_data(), ours.glyph_data());
+        assert_eq!(orig_bytes.len(), bytes.len());
+        assert_eq!(orig_bytes.as_ref(), bytes);
+    }
+
```

Comment:
Shouldn't we have a test for the flag repeat change too? - the test matches the PR title but the PR includes that additional change.